src/main/resources/db/changesets/20190117_createddates_and_multicolumn.yaml

```diff
@@ -0,0 +1,66 @@
+databaseChangeLog:
+  - changeSet:
+      id: created_dates
+      tagDatabase:
+        tag: created_dates
+      author: jhert
+      changes:
+        - addColumn:
+            tableName: study
+            columns:
+              - column:
+                  name: created_date
+                  type: timestamptz
+                  defaultValueComputed: now()
+                  constraints:
+                    nullable: false
+
+        - addColumn:
+            tableName: asset_specification
+            columns:
+            - column:
+                name: created_date
+                type: timestamptz
+                defaultValueComputed: now()
+                constraints:
+                  nullable: false
+
+        - dropColumn:
+            tableName: study_relationship
+            columnName: from_column
+
+        - dropColumn:
+            tableName: study_relationship
+            columnName: to_column
+
+        - createTable:
```

Comment:
I think you will need some sequence number column in here. The order of the columns is significant:
(id, timestamp) is different from (timestamp, id).

Reply:
I toyed around with using an array but as far as I can tell there is no way to define a foreign key on items in the array. I'd prefer to get the referential integrity guarantees we get from the FKs so I added a sequence_num column instead.

Reply:
(i know we talked, but commenting here also). i thought we had decided we didn't need to support multicolumn relationships initially